network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_index_featured_video_post.html

```diff
@@ -0,0 +1,43 @@
+{% load wagtailcore_tags wagtailimages_tags i18n localization homepage_tags blog_tags static %}
+
+  <div class="tw-grid tw-grid-cols-12 tw-mb-7">
+    <div class="tw-pr-[36px]  tw-py-3  tw-flex tw-flex-col tw-col-span-12 large:tw-col-span-4 large:tw-py-0">
+      <div class="tw-flex tw-flex-wrap">
+        {% with topics=blog_page.topics.all topic_classes="tw-h6-heading tw-text-blue-80 tw-mr-2 tw-my-1" %}
```

Comment:
Follow up ticket to componentize this: https://github.com/mozilla/foundation.mozilla.org/issues/8980

Reply:
Yea good call 👍